### ISSUE TITLE
Allow for keyboard echo settings.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#131]
```
What steps will reproduce the problem?
1. open any edit field.
2. start typing.
3. you hear feedback

What is the expected output? What do you see instead?
there should be a way to change the amount of feedback for typing.

What version of the product are you using? On what operating system?
android 4.1.1 with the included Talk back

Please provide any additional information below.


```

Original issue reported on code.google.com by `jmwalto...@gmail.com` on 9 Sep 2012 at 6:33

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 13 Sep 2012 at 6:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Unless I'm not understanding it completely, an area of the page says 
it's for jellybean. Does this mean these changes will only work for jb 
or also on ics as well.
```

Original comment by `icu8...@gmail.com` on 13 Sep 2012 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
Keyboard echo settings are included in 3.2.1.
```

Original comment by `caseybur...@google.com` on 4 Dec 2012 at 1:54
- Changed state: **Fixed**
